scripts/msg.h.template

```diff
@@ -55,6 +55,17 @@ for field in spec.parsed_fields():
       package = package or spec.package # convert '' to package
       print('#include <%s/%s.h>'%(package, name))
 }@
+@##############################
+@# Plugins
+@##############################
+@{
+import os;
+if os.path.exists('include/%s/plugin/%s.h'%(spec.package, spec.short_name)):
+  have_plugin = True
+  print('#include <%s/plugin/%s.h>'%(spec.package, spec.short_name));
+else:
+  have_plugin = False
+}@
```

Comment:
Nitpick: rewrite as:

```
@{
import os
has_plugin = os.path.exists('include/%s/plugin/%s.h' % (spec.package, spec.short_name))
}@
```

and then use it the same way as for the other macros:

```
@[if has_plugin]@
#include <@(spec.package)/plugin/@(spec.short_name).h>
@[end if]@
```